communication/src/main/java/org/jboss/da/communication/pom/GalleyWrapper.java

```diff
@@ -49,6 +53,8 @@
 
     private final LocalRepo localRepo;
 
+    private Configuration config = new Configuration();
```

Comment:
I don't think it's a good idea to create a been by new. Now it works, but if we change Configuration so it Injects anything, it would break.

I'm not sure what is the best way to approach this, I have two ideas:
 * create field ``String indyReporitory`` and create the URL in constructor (where you add a ``Configuration`` paramenter)
 * add ``Configuration`` paramenter to ``addLocations`` method.